mne_connectivity/spectral/tests/test_spectral.py

```diff
@@ -408,6 +408,137 @@ def test_spectral_connectivity(method, mode):
     assert (out_lens[0] == 10)
 
 
+@pytest.mark.parametrize('method', ['mic', 'mim', 'gc', 'gc_tr',
+                                    ['mic', 'mim', 'gc', 'gc_tr']])
+@pytest.mark.parametrize('mode', ['multitaper', 'fourier', 'cwt_morlet'])
+def test_multivar_spectral_connectivity(method, mode):
+    """Test multivariate frequency-domain connectivity methods."""
```

Comment:
Is this mainly a test to catch errors that should be raised? If so, can we name the test and add a note in the docstring accordingly?

Reply:
At the moment, yes, this is really just catching errors (since much of the behaviour is already being tested in `test_spectral_connectivity`). I will update the name and description.